custom/ewsghana/tests/test_webusers_sync.py

```diff
@@ -57,6 +57,7 @@ def test_create_webuser(self):
         self.assertEqual(TEST_DOMAIN, domain_name)
         self.assertEqual(UserRole.get_read_only_role_by_domain(TEST_DOMAIN)._id,
                          ewsghana_webuser.get_domain_membership(TEST_DOMAIN).role_id)
+        ewsghana_webuser.delete()
```

Comment:
good point. it's often a tradeoff between something like this and doing a giant hammer to delete everything in setup/teardown (which typically makes the tests slower and is something I've been trying to avoid recently)

i think since our CI runs with --failfast it's probably ok in most instances, even if not great. try/finally is probably the best approach, but agree not obviously worth it every time.